### PR TITLE
Dzelge init cate docker image

### PR DESCRIPTION
**Pull request by dzelge**
This PR creates a Dockerfile that configures a base cate docker image.

### CODE REVIEW

File: .github/workflows/quay.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ on:
     branches:
       - master
       - stage_*
+      - v*
```
Comment (forman) [inline .github/workflows/quay.yml]: Please comment, I'm not sure I understand correctly.

File: .github/workflows/quay.yml [hunk 2]
```diff
@@ -29,6 +29,7 @@ on:
     branches:
       - master
       - stage_*
```
Comment (forman) [inline .github/workflows/quay.yml]: Please comment, I'm not sure I understand correctly.

**Review by forman — COMMENTED** [review]
Thanks!

File: .github/workflows/quay.yml (hunk 1 above)
Comment (dzelge) [reply .github/workflows/quay.yml]: I removed this as tghis was a test of GitHub actions.

File: .github/workflows/quay.yml (hunk 2 above)
Comment (dzelge) [reply .github/workflows/quay.yml]: removed. Was a test of GitHub actions

**Pull request merged**